platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu_provisioning.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=dpu config db provisioning
+Requires=docker.service
+After=docker.service
+After=dpu.service
+Requires=dpu.service
+Before=database.service
+
+[Service]
+User=root
+ExecStartPre=/bin/sleep 20
```

Comment:
Please remove sleep here if it's not needed. If it's needed, please determine what it is actually waiting on and add a check for that instead.

Reply:
done